waltz-test/src/main/python/waltz_ducktape/tests/validation/load_rebalancing_test.py

```diff
@@ -62,7 +61,8 @@ def rebalancing_on_server_nodes(self, validation_cmd, timeout, num_active_partit
         server_node = self.waltz_server.nodes[partition_future_server_index]
         server_node_hostname = server_node.account.ssh_hostname
         server = self.get_host(server_node_hostname, server_port)
-        self.server_cli.add_preferred_partition(server, random_active_partition)
+        server_cli = ServerCli(server_node, self.cli_config_path)
```

Comment:
Curious why do we move the ServerCli initialization from the constructor to here and use a different way of fetching server node? Other than this question the PR looks good to me

Reply:
This code change is just to enforce that the serverCli command is executed from one of the waltz-server VM instead of a client VM (as in the constructor we are using verifiable_client.nodes[0]) and with the correct config parser. 

So, that way if executed from the waltz-server VM, it will have the ssl information parsed as expected (i.e. in "ssl." format). Whereas the client VMs have ssl information in their config file parsed in "client.ssl" format.

Reply:
We are not modifying the way waltz-server node is fetched. Its still the same (via line 63).